nyaa/templates/rss.xml

```diff
@@ -40,6 +40,7 @@
 			<nyaa:categoryId>{{- cat_id }}</nyaa:categoryId>
 			<nyaa:category>  {{- category_name(cat_id) }}</nyaa:category>
 			<nyaa:size>      {{- torrent.filesize | filesizeformat(True) }}</nyaa:size>
+			<nyaa:comments>      {{- torrent.comment_count }}</nyaa:comments>
```

Comment:
You're now including the comment count from the other PR!

Reply:
I'm not even sure how did it end up here, ehh.